profiles/ApexLegends/index.js

```diff
@@ -0,0 +1,56 @@
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+
+export const appID = 1172470;
+export const title = "Apex Legends";
+
+const playingRegex = /^(?:(.*) - (.*)(\(.*\))$|(.*) - (.*))$/;
+
+export function translateSteamPresence(steamRichPresence) {
+    let dir = dirname(fileURLToPath(import.meta.url));
+    let discordRichPresence = {};
+    let resources = null;
+    discordRichPresence.details = "Playing Apex Legends";
+    discordRichPresence.state = steamRichPresence;
+
+    if(!fs.existsSync(dir + '/resources.json')) {
+        return discordRichPresence;
+    }
+
+    resources = JSON.parse(fs.readFileSync(dir + '/resources.json'));
```

Comment:
Is it possible to replace the entire resources.json loading/parsing with an import? Node allows you to use `import resources from "./resources.json;" which would remove a lot of the code needed and would also provide an error if the file isn't there which is nice.

I also question if you should load the resources from disk each time this function is called. It's kinda wasteful unless you are debugging the app and wanna live-edit the resources.json.